tox.ini

```diff
@@ -1,13 +1,22 @@
 [tox]
 envlist = py27
 
+[pylama]
+format = mccabe,pep8,pep257,pyflakes,pylint
+ignore = C901
+
+[pylama:pep8]
+max_line_length = 100
```

Comment:
I would suggest that we move this again and a better place is definitely `setup.cfg`. I figured that out while checking other repositories like https://github.com/davehunt/PyPOM. As you can see there the tox.ini is already complex and we do not want to make it even harder to read.